relativedeltafield/__init__.py

```diff
@@ -25,9 +25,31 @@
     r'$'
 )
 
+class iso8601relativedelta(relativedelta):
```

Comment:
What is the purpose of this class?  It seems to do nothing but pass its arguments on to the superclass, and optionally remember the original value passed in.

Reply:
I thought the str() would be better to return an is88601 string format rather than "relativedelta(...)"